bin/make-deb

```diff
@@ -11,10 +11,10 @@ def main():
         debconf = DebianConfiguration(os.getcwd())
         debconf.render()
     except DebianConfigurationException as e:
-        print e
+        print(e)
```

Comment:
You need to add `from __future__ import print_function` as the first line, otherwise this can have surprising behavior when modified.